src/app/people/page.tsx

```diff
@@ -1,5 +1,229 @@
+    })};
+  padding-right: ${linearlyScaleSize({
+    minSizePx: 24,
+    maxSizePx: 8,
+    minScreenSizePx: parseInt(ScreenSize.sm),
+    maxScreenSizePx: parseInt(ScreenSize.md),
+  })};
+  width: 100%;
+  gap: 16px;
+  @media (min-width: ${ScreenSize.max}) {
+    width: ${ScreenSize.max};
+    margin: 0 auto;
+  }
+
+  & > main {
+    // make the main content (publications list) take up 85% and the Sidebar component 15%
+    width: 85%;
+  }
+`
+const SideContainer = styled.div`
+  padding-top: 96px;
+  padding-left: 30px;
+  padding-right: 30px;
+  width: 15%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  @media (max-width: ${ScreenSize.sm}) {
+    display: none;
+  }
+`
+const AlumniSectionContent = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+  gap: 1px;
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(5, 1fr);
+  }
+`
+
+const SubCategoryTitle = styled.h2`
+  ${FontVariant.title_sm}
+`
+
+const SpecialThanksSection = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 16px;
+`
+const SpecialThanksTitle = styled.h2`
+  font-size: 1.5rem;
+  color: ${Color.gray900};
+`
+
+const SpecialThanksText = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const SpecialThanksSubtitle = styled.h3`
+  ${FontVariant.body_lg}
+  color: ${Color.gray900};
+  margin: 0;
+`
+
+const SpecialThanksDescription = styled.p`
+  ${FontVariant.body_md}
+  color: ${Color.gray700};
+  gap: 4px;
+`
 
 export default function Page() {
-  return <main></main>
+  const [selected, setSelected] = useState('')
+  const [sidebarList, setSidebarList] = useState<string[]>([])
+  const [activeSection, setActiveSection] = useState<string | null>(null)
+  const ignoreObserver = useRef(false)
+  const sectionRefs = useRef<Record<string, HTMLElement | null>>({})
+
+  useEffect(() => {
+    const observerCallback = (entries: IntersectionObserverEntry[]) => {
+      if (ignoreObserver.current) return
+
+      entries.forEach(entry => {
+        if (entry.isIntersecting) {
+          setSelected(entry.target.id)
+          setActiveSection(entry.target.id)
+        }
+      })
+    }
+
+    const observer = new IntersectionObserver(observerCallback, { threshold: 0.1 })
+    const sections = Object.values(sectionRefs.current)
+    sections.forEach(section => {
+      if (section) {
+        observer.observe(section)
+      }
+    })
+
+    return () => {
+      sections.forEach(section => {
+        if (section) {
+          observer.unobserve(section)
+        }
+      })
+    }
+  }, [])
+
+  useEffect(() => {
+    handleSidebarList(MEMBERS_KEY)
+  }, [MEMBERS_KEY])
+
+  const handleSidebarList = (list: string[]) => {
+    const sections = CareerTypes.filter(career => career !== 'Alumni').map(career =>
+      career.replace(/\s+/g, '').replace('.', '')
+    )
+    sections.push('alumni')
+    setSidebarList(sections)
```

Comment:
Is there a reason why you formatted the text to have no whitespace in the Sidebar?

Reply:
![Uploading Screenshot 2024-07-15 at 2.27.47 PM.png…]()